src/Dropdown.php

```diff
@@ -177,10 +177,15 @@ public static function show($itemtype, $options = [])
         }
 
         if ($params['readonly']) {
-            return '<span class="form-control" readonly'
+            $output = '';
+            foreach ($names as $name) {
+                $output .= "<input type='hidden' name='".$param ['name']."' value='$name'>";
+            }
```

Comment:
I may be wrong, but the expected value is probably `$params['value']` (or `$params['values']` when `$params['multiple']` is true).
Also, a `[]` should probably be added to `$param['name']` when `$params['multiple']` is true.

Reply:
You're right for `[]` in the name.

`$params['values']` are computed into `$names` a few lines before, but maybe is that only relevant for display? I have no idea, I have aslo no idea how to reproduce the original issue.

Reply:
I've change both, `$names` is probably only computed for display.